app/addons/documents/base.js

```diff
@@ -42,14 +42,18 @@ function getQueryParam (query) {
   return query;
 }
 
+function partitionUrlComponent(partitionKey) {
+  return partitionKey ? '/_partition/' + partitionKey : '';
+}
+
 FauxtonAPI.registerUrls('allDocs', {
   server: function (id, query) {
     /** XXX DEPRECATED: use allDocsSanitized **/
     return Helpers.getServerUrl('/' + id + '/_all_docs' + getQueryParam(query));
   },
-  app: function (id, query) {
+  app: function (id, partitionKey, query) {
     /** XXX DEPRECATED: use allDocsSanitized **/
```

Comment:
Are these truly deprecated. If so we should remove them and use the correct ones. Or remove the comment.

Reply:
It's still used in several places so I'm removing the comment